package.json

```diff
@@ -196,10 +196,10 @@
     "@atlaskit/pragmatic-drag-and-drop-hitbox": "^1.1.0",
     "@floating-ui/dom": "~1.2.9",
     "@floating-ui/react-dom": "~1.3.0",
-    "@lexical/headless": "^0.21.0",
-    "@lexical/link": "^0.21.0",
-    "@lexical/react": "^0.21.0",
-    "@lexical/selection": "^0.21.0",
+    "@lexical/headless": "^0.38.2",
```

Comment:
Comment: as lexical are not following semver for the pre alpha v1 releases, and can/will include breaking changes in any release before then, I think we should pin these versions. Even though `npm i` will not bump 0 versioned it will if `update` is run or someone does a ful install after deleting the lock file etc